zebrium/manifest.json

```diff
@@ -0,0 +1,67 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "a1fa9510-af05-4950-ad67-4eed3f14d4bf",
+  "app_id": "zebrium",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows",
+    "Category::Notification",
+    "Offering::UI Extension",
+    "Offering::Integration"
+  ],
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "When you know there's a problem, see the reason on your dashboard",
```

Comment:
I'd recommend making this a bit more descriptive as to what Zebrium does specifically

Reply:
Done